Formula/openjdk@11.rb

```diff
@@ -87,17 +100,32 @@ def install
       --with-jvm-variants=server
     ]
 
-    if Hardware::CPU.arm?
-      args += %W[
-        --disable-warnings-as-errors
-        --openjdk-target=aarch64-apple-darwin
-        --with-build-jdk=#{boot_jdk}
-        --with-extra-cflags=-arch\ arm64
-        --with-extra-ldflags=-arch\ arm64\ -F#{framework_path}\ -headerpad_max_install_names
-        --with-extra-cxxflags=-arch\ arm64
-      ]
-    else
-      args << "--with-extra-ldflags=-headerpad_max_install_names"
+    on_macos do
```

Comment:
```suggestion
    framework_path = nil
    on_macos do
```
You still need this so that `framework_path` lives on after the current `on_macos` block.